### PR TITLE
Docs: add instructions for openSUSE 15.0

### PR DESCRIPTION
**Pull request by mbologna**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -263,6 +263,15 @@ $ sudo dnf copr enable carlwgeorge/ripgrep
 $ sudo dnf install ripgrep
 ```
 
+If you're an **openSUSE Leap 15.0** user, you can install ripgrep from the
+[utilities repo](https://build.opensuse.org/project/show/utilities/ripgrep):
```
Comment (BurntSushi) [inline README.md]: I get a 404 error for this.

File: README.md [hunk 2]
```diff
@@ -263,6 +263,15 @@ $ sudo dnf copr enable carlwgeorge/ripgrep
...
+
+```
+$ sudo zypper ar https://download.opensuse.org/repositories/utilities/openSUSE_Leap_15.0/utilities.repo
```
Comment (BurntSushi) [inline README.md]: Does this need to be updated for every new major release of openSUSE?

File: README.md [hunk 3]
```diff
@@ -263,6 +263,15 @@ $ sudo dnf copr enable carlwgeorge/ripgrep
 $ sudo dnf install ripgrep
 ```
 
+If you're an **openSUSE Leap 15.0** user, you can install ripgrep from the
```
Comment (BurntSushi) [inline README.md]: It looks like we already have openSUSE Tumbleweed instructions. Is there any way to collapse them into one section? What's the difference? (I don't know what the difference between them is.

Comment (mbologna) [reply README.md]: Leap and Tumbleweed are very different: the difference is verbosely explained in https://en.opensuse.org/Portal:Tumbleweed and https://www.opensuse.org/#Leap

Maybe we can create a super section called "openSUSE" and then introduce the two flavors to get ripgrep

File: README.md (hunk 2 above)
Comment (mbologna) [reply README.md]: Yes, this depends on the release

File: README.md (hunk 1 above)
Comment (mbologna) [reply README.md]: You are right. Fixed

**Pull request merged**